crates/yas_core/src/libraries/sqrt_price_math.cairo

```diff
@@ -203,4 +242,45 @@ mod SqrtPriceMath {
             )
         }
     }
+
+    fn check_sqrtPX96_sign(sign: bool) -> Result<(), felt252> {
+        if sign == false {
+            Result::Ok(())
+        } else {
+            Result::Err('sqrt_ratio_AX96 cannot be neg')
+        }
```

Comment:
Every operation costs gas, so we should minimize unnecessary operations to save on gas.
```suggestion
        if sign {
            Result::Err('sqrt_ratio_AX96 cannot be neg')
        } else {
            Result::Ok(())
        }
```